### ISSUE TITLE
otto dev ignores Vagrantfile in project root

### ISSUE THREAD

**Issue by Comradin** [hashicorp/otto#173]
Hello,

I have a custom app and defined an Appfile like described in the documentation:

    customization "dev" {
        vagrantfile = "Vagrantfile"
        box = "jenkins"
    }
    
    customization "dev-dep" {
        vagrantfile = "Vagrantfile"
        box = "jenkins"
    }

As you can see, I have a vagrant.vm.define(:jenkins) block in my Vagrantfile located in the project root.

Now I issue an otto compile && otto dev and guess what, the default hashicorp/precise64 box gets started. My Vagrantfile is getting silently ignored. It does not matter if I define the path like "Vagrantfile", "relative/path/to/Vagrantfile", or "/full/path/to/Vagrantfile". It's not getting used.

When I look into the .otto directory structure I see there are two "Vagrantfiles". One in .otto/compiled/app/dev/ and one Vagrantfile.fragment in .otto/compiled/app/dev-dep. The one in dev is being used, the one in dev-dep contains my Vagrantfile, my box, everything I have configured. But it's being ignored.

    # otto dev vagrant up jenkins
    ==> Executing: 'vagrant up jenkins'
    The machine with the name 'jenkins' was not found configured for
    this Vagrant environment.
    
    Error executing Vagrant: exit status 1
    
    The error messages from Vagrant are usually very informative.
    Please read it carefully and fix any issues it mentions. If
    the message isn't clear, please report this to the Otto project.

Even otto dev vagrant status does not know the jenkins box:
    otto dev vagrant status
    ==> Executing: 'vagrant status'
    Current machine states:
    
    default                   running (virtualbox)

Whereas a *vagrant status* in the project root does know it very well:
    vagrant status
    Current machine states:
    
    jenkins                   not created (virtualbox)

Any hints how to fix this? What am I missing or where have I gone wrong?

Regards,
Marcus

**Comment by Comradin** [comment]
Ok,
I just discovered that my Vagrantfile is being included in the compiled/app/dev/Vagrantfile, but the box definition is not used as if the Vagrantfile is kind of invalid, but as there is no error message Im a bit puzzled.

**Comment by ilyar** [comment]
:+1

**Comment by RitchieFlick** [comment]
I can confirm this behavior on a Windows 8.1 machine (Powershell).

**Comment by bravenewprince** [comment]
Same problem on my machine. OSX Otto 0.1.1

**Comment by cromulus** [comment]
Yes, I'm running out of memory in my dev environment and would like to increase the memory for the base VM. I tried creating a Vagrantfile in the root of my project that only defined the memory size but it's not being included in Otto's compiled Vagrantfile.
